mantidproject/mantid
# Remove python 2 __future__ Imports and Imports of six

**Description of work.**
- Removes all `from __future__ import` statements.
- Replaces all six imports and usages with python 3 versions.
- Removes python version checks.

**To test:**
- Check all python interfaces open
- Check plotting works as expected.

Resolves part of #27722 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
